pkg/distro/defs/fedora/distro.yaml

```diff
@@ -1494,12 +1494,12 @@ image_types:
     # this is the eventual name, and `wsl` the alias but we've been
     # having issues with CI renaming it
     name_aliases: ["server-wsl"]
-    filename: "wsl.tar"
+    filename: "image.wsl"
```

Comment:
In the rhel `mkWSLImgType()` the filename is `disk.tar.gz` - maybe worth a small comment why we diverge in the various place. Also will anything break if we change the filename? I.e. do our users expect the old one? Do we need to release-note this somehow?

Reply:
Since the previous WSL images for Fedora never worked at all (they cannot be imported, they were in an incorrect format) I don't suspect this to break for any users.

The reason for the name change and link to Microsoft documentation is listed in the commit message. WSLv2 images *must* have the `.wsl` file extension.